docs/advanced/configure-cf-external-databases-using-azure-mysql-postgres-service/migrate-mysql-databases/README.md

```diff
@@ -0,0 +1,97 @@
+# Migrate Cloud Foundry MySQL databases to Azure Database for MySQL
+
+On Cloud Foundry (CF), it is recommended to use PAAS service for the purpose of high availability. [Azure Database for MySQL](https://azure.microsoft.com/en-us/services/mysql/) is one of services that CF can levege for HA. 
+
+For a new deployment, you can refer to this [guidance](../README.md) to use Azure Database for MySQL as external databases.
+For an existed deployment, if you want to use Azure Database for MySQL, this document will provide a simple guildance to migrate the database.
+
+**NOTE:** Be careful on operating databases, you should practise on a test environment before officially do the migration on your production environment.
+
+In this guidance, we will use `mysqldump` and `mysql` command-lines to dump backup files from internal databases and then retore them to Azure Databases for MySQL.
+
+1. Figure out databases to be migrated
+
+    Check your CF manifest carefully, find out which databases are used. Write down database name, user, password for  step `Migrate databases`. If you are using Ops Manager, follow this [guid](https://docs.pivotal.io/pivotalcf/2-2/customizing/credentials.html) to get credentials.
+    
+    Here is a list sample of the databases used in CF.
+    ```
+        +----------------------+
+        | Database             |
+        +----------------------+
+        | cloud_controller     |
+        | diego                |
+        | locket               |
+        | network_connectivity |
+        | network_policy       |
+        | routing-api          |
+        | uaa                  |
+        +----------------------+
+    ```
+
+1. Prepare Azure Database for MySQL
+
+    * Follow guidance on this [link](../README.md#prepare-azure-mysqlpostgres-database), please remember to turn on `Allow access to Azure services` and Disable `Enforce SSL connection in Azure MySQL`.
+    * Follow guidance on this [link](../README.md#create-the-databases) to create releated databases.
+
+1. Stop CC VMs
+
+    * Turn off bosh resurrector (`bosh -e azure -d cf  update-resurrection off`)
+    * On Azure portal, stop all the CC VM jobs (Cloud Controller, Cloud Controller Worker, and Clock Global). Note that your CF API services will stop being available at this point (running apps should continue to be available though)
```

Comment:
change from "stop all the CC VM jobs" to "stop all the vms which the cc jobs are running on"?

Reply:
copied from https://github.com/pivotal-cf/goblob/blob/master/README.md#removing-nfs-post-migration. But I think your wording is better, will change.